.github/workflows/lint.yaml

```diff
@@ -26,3 +25,27 @@ jobs:
           output: ' '
       - name: Lint modified files
         run: pre-commit run --files ${{ steps.file_changes.outputs.files }}
+  type_checking:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+      - name: Setup conda env
```

Comment:
N00b Q: do we need to install the library in order to run mypy? Is it possible to just check out the python interpreter, source code and run on the PR changes (like in linter)?

Reply:
it complains about imports if you don't do installation of torch or might happen with other dependencies (repros locally too)